sdk/storage/azure-storage-blob/tests/test_get_blob.py

```diff
@@ -1019,4 +1019,22 @@ def test_get_blob_progress_readinto(self, storage_account_name, storage_account_
         progress.assert_complete()
         self.assertEqual(len(data), read)
 
+    @BlobPreparer()
+    def test_unicode_get_blob_binary_data2(self, storage_account_name, storage_account_key):
```

Comment:
What's this test for? I don't think we need this?

If we do need it, you'll need to check in a recording.

Reply:
oops